suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -490,6 +491,7 @@ export const networks = {
         features: ['tokens', 'staking'],
         explorer: {
             tx: 'https://testnet-explorer.blockfrost.dev/transaction/',
```

Comment:
We should probably replace these as well with https://preview.cexplorer.io/...
Some of the URLs are not working for me

Reply:
5af2707436895a0a351002a5512222529275825f